modules/cnf-topology-aware-lifecycle-manager-about-cgu-crs.adoc

```diff
@@ -33,218 +45,373 @@ After {cgu-operator} completes a cluster update, the cluster does not update aga
-The periodic checks allow the updates to complete if they get prolonged due to network, CPU, or other issues. 
+* `true`
++
+All clusters are compliant with the specified managed policies.
+* `false`
++
+Policy remediation failed as there were no clusters available for remediation, or because policy remediation took too long for one of the following reasons:
++
+** The current batch contains canary updates and the cluster in the batch does not comply with all the managed policies within the batch timeout.
+** Clusters did not comply with the managed policies within the `timeout` value specified in the `remediationStrategy` field.
 
-{cgu-operator} transitions to the `UpgradeTimedOut` state in two cases:
 
-* When the current batch contains canary updates and the cluster in the batch does not comply with all the managed policies within the batch timeout.
-* When the clusters do not comply with the managed policies within the `timeout` value specified in the `remediationStrategy` field.
+.Sample `ClusterGroupUpgrade` CR in the `Succeeded` state
 
-If the policies are compliant, {cgu-operator} transitions to the `UpgradeCompleted` state.
-
-[id="upgrade_completed"]
-== The UpgradeCompleted state
+[source,yaml]
+----
+    apiVersion: ran.openshift.io/v1alpha1
+    kind: ClusterGroupUpgrade
+    metadata:
+      name: cgu-upgrade-complete
+      namespace: default
+    spec:
+      clusters:
+      - spoke1
+      - spoke4
+      enable: true
+      managedPolicies:
+      - policy1-common-cluster-version-policy
+      - policy2-common-pao-sub-policy
+      remediationStrategy:
+        maxConcurrency: 1
+        timeout: 240
+    status:
+      clusters:
+        - name: spoke1
+          state: complete
+        - name: spoke4
+          state: complete
+      conditions:
+      - message: All selected clusters are valid
+        reason: ClusterSelectionCompleted
+        status: "True"
+        type: ClustersSelected <1>
+      - message: Completed validation
+        reason: ValidationCompleted
+        status: "True"
+        type: Validated <2>
+      - message: All clusters are compliant with all the managed policies
+        reason: Completed
+        status: "False"
+        type: Progressing <3>
+      - message: All clusters are compliant with all the managed policies
+        reason: Completed
+        status: "True"
+        type: Succeeded <4>
+      managedPoliciesForUpgrade:
+      - name: policy1-common-cluster-version-policy
+        namespace: default
+      - name: policy2-common-pao-sub-policy
+        namespace: default
+      remediationPlan:
+      - - spoke1
+      - - spoke4
+      status:
+        currentBatchRemediationProgress:
+          spoke4:
+            state: Completed <5>
+----
+<1> The `ClustersSelected` fields show that all selected clusters are valid.
+<2> The `Validated` fields show that all selected clusters have been validated.
```

Comment:
Not sure if we need to repeat these two in every example.

Reply:
ok - removed